metricbeat/module/docker/memory/helper.go

```diff
@@ -0,0 +1,56 @@
+package memory
+
+import (
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/metricbeat/module/docker"
+	"github.com/elastic/beats/libbeat/logp"
+)
+
+type MEMORYData struct {
+	Time        common.Time
+	MyContainer *docker.Container
+	Failcnt     uint64
+	Limit       uint64
+	MaxUsage    uint64
+	TotalRss    uint64
+	TotalRss_p  float64
+	Usage       uint64
+	Usage_p     float64
+}
+type MEMORYService struct {
+}
+
+func NewMemoryService() *MEMORYService {
+	return &MEMORYService{}
+}
+
+func (c *MEMORYService) GetMemorystatsList(rawStats []docker.DockerStat) []MEMORYData {
+	formatedStats := []MEMORYData{}
+	if len(rawStats) != 0 {
+		for _, myRawStats := range rawStats {
+			formatedStats = append(formatedStats, c.getMEMData(myRawStats))
+		}
+	} else {
+		logp.Info("No container is running \n")
```

Comment:
When logging messages you can omit the `\n`. It is added by the logging framework.